### ISSUE TITLE
Authentication Error: JWT is invalid when changing Auth0 links in GoLang/ReactJS app

### ISSUE THREAD

**Issue by ShaalanMarwan** [vlab-research/vlab#216]
I am encountering an authentication issue when attempting to change the Auth0 links in the GoLang/ReactJS app. After following the provided .env-example and creating my own .env file with the necessary configuration, the backend consistently returns a ["JWT is invalid"](https://github.com/vlab-research/vlab/blob/081145d8c605f7a4c4290cf8552abdb6629bcef9/api/internal/server/middleware/auth/auth.go#LL63C4-L63C4) error. 

When I tried to hardcoded these [two links](https://github.com/vlab-research/vlab/blob/081145d8c605f7a4c4290cf8552abdb6629bcef9/api/internal/server/server.go#L44) and it worked.

**Comment by Spazzy757** [comment]
@ShaalanMarwan did you update both the env in the [Api](https://github.com/vlab-research/vlab/blob/main/api/.env-example) as well as the [Dashboard](https://github.com/vlab-research/vlab/blob/main/dashboard/.env-example) directory?

**Comment by ShaalanMarwan** [comment]
@Spazzy757 Yes definitely, cuz when I hardcoded them, they started working normally

**Comment by Spazzy757** [comment]
If you only changed it in the one place that you mentioned then the frontend would not work. So did you also hardcode it in the UI (Dashboard Directory)?

**Comment by ShaalanMarwan** [comment]
On the frontend side, I only changed in `.env ` but in the backend I hardcoded it

**Comment by Spazzy757** [comment]
In the line that you mentuioned hard coding, can you log out what the value for `cfg.Auth0` is

**Comment by ShaalanMarwan** [comment]
it shows me the domain and Audience link
